src/pages/AssetPage/index.tsx

```diff
@@ -104,6 +104,16 @@ export class AssetPage extends React.Component<AssetPageProps> {
         <TransactionsTable
           transactions={this.transactionStore.assetTransactions}
           loadTransactions={this.loadTransactions}
+          stickyTitle={
+            <div className="sticky-title">
```

Comment:
I think it makes sense to move this title to separate component or, at least, return it using other render function